src/core/core_timing.cpp

```diff
@@ -176,7 +176,8 @@ u64 Timing::Timer::GetTicks() const {
 }
 
 void Timing::Timer::AddTicks(u64 ticks) {
-    downcount -= static_cast<u64>(ticks * cpu_clock_scale);
+    downcount -=
+        static_cast<u64>((Settings::values.raise_cpu_ticks ? 16000 : ticks) * cpu_clock_scale);
```

Comment:
This code seems to *set* the number of ticks to 16000 rather than *adding* 16000 ticks. Is this intended?

Reply:
Yeah that's how It works

Reply:
In that case the description of the option seems slightly inaccurate
It's less-so adding 16000 ticks and moreso hardcoding the number of added ticks to always be 16000

Reply:
Yeah I edited the desc to be a bit more accurate. Should be fine now